### PR TITLE
Nodejs week1

### PR DESCRIPTION
**Pull request by slastushevskyi**

### CODE REVIEW

File: nodejs/week1/avg.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  // if it is Number or not. Also it helps us to sort out
+  // string values
+  if (!isNaN(parseInt(process.argv[i]))) {
+    n += parseInt(process.argv[i]);
+    m += 1;
+  }
+}
+// if no arguments are provided
+if (isNaN(n / m)) {
+  console.log("Please enter some numbers");
```
Comment (veronicaburduzhan) [inline nodejs/week1/avg.js:17]: Function still executes and calculates average number even when I enter a string as an argument.

**Review by veronicaburduzhan — APPROVED** [review]
Hey, your function is calculating the average of the numbers but it doesn't work if the provided arguments are not numbers :(

File: nodejs/week1/avg.js (hunk 1 above)
Comment (slastushevskyi) [reply nodejs/week1/avg.js:17]: No, it works :) It showing the error message if user provided only strings or an empty string.
If there are in input both number and strings - it shows average of numbers. And of course average of numbers if user will provide only numbers :)

Comment (veronicaburduzhan) [reply nodejs/week1/avg.js:17]: That's what I’m saying: if I enter both strings and numbers function still executes. But it probably should gives en error to provide all valid numbers, don't you think?😊

**Comment by slastushevskyi** [comment]
In my case it handle only numbers😅
I understand what you mean, but optional task says: what if user enter a
string. It doesn’t saying that it should show an error, I just to handle
this case somehow (imo).
So my program executes an average number

lør. 25. mar. 2023 kl. 06.16 skrev Veronica Burduzhan <
***@***.***>:

**Pull request merged**